Cabal/Cabal.cabal

```diff
@@ -341,16 +314,16 @@ test-suite package-tests
     tasty-quickcheck,
     tasty-hunit,
     QuickCheck >= 2.1.0.1 && < 2.9,
+    transformers,
```

Comment:
It's fine because it's a test dep. Otherwise the deps on tasty/co would be unacceptable.